lib/core/collections.toit

```diff
@@ -3514,9 +3552,11 @@ class Deque extends List implements Collection:
   */
   remove --at/int -> any:
     last := size - 1
-    if at < 0 or at > last: throw "OUT_OF_BOUNDS"
-    if at >= last >> 1:
-      return backing_.remove --at=(first_ + at)
+    if not 0 <= at <= last: throw "OUT_OF_BOUNDS"
+    if at >= (last >> 1):
```

Comment:
Add a comment here?

Reply:
done. And refactored it to make it easier to follow.